.circleci/config.yml

```diff
@@ -3,54 +3,88 @@ version: 2.1
 orbs:
   codacy: codacy/base@1.0.1
 
-jobs:
-  deploy_to_doks:
+references:
+
+  default_doks_image: &default_doks_image
     docker:
       - image: codacy/ci-do:0.2.3
-    environment:
-      DOKS_CLUSTER_NAME: codacy-doks-cluster-dev
     working_directory: ~/workdir/
+
+    # doks instance types can be found here https://developers.digitalocean.com/documentation/v2/
+  release_environment: &release_environment
+    DOKS_CLUSTER_NAME: codacy-doks-cluster-release
+    DO_TF_WORKSPACE: release
+    NODE_TYPE: s-1vcpu-3gb
+    K8S_VERSION: 1.15
+
+  nightly_environment: &nightly_environment
+    DOKS_CLUSTER_NAME: codacy-doks-cluster-nightly
+    DO_TF_WORKSPACE: nightly
+    NODE_TYPE: s-1vcpu-3gb
+    K8S_VERSION: 1.15
+
+  hourly_environment: &hourly_environment
+    DOKS_CLUSTER_NAME: codacy-doks-cluster-hourly
+    DO_TF_WORKSPACE: hourly
+    NODE_TYPE: s-1vcpu-3gb
+    K8S_VERSION: 1.15
+
+  dev_environment: &dev_environment
```

Comment:
Left k8s version and node type untouched. these are the values that are in the current dev cluster